cluster/gce/util.sh

```diff
@@ -868,7 +868,7 @@ function setup-monitoring-firewall {
   done
   if [[ ${grafana_host} != *"<"* ]]; then
```

Comment:
+1 to what @zmerlynn said. Since we don't use the grafana_host anymore, that logic can be deleted.